website/source/docs/guides/creating-certificates.html.md

```diff
@@ -296,10 +305,15 @@ Binding to `0.0.0.0` should work:
 ```json
 {
   "ui": true,
-  "client_addr": "0.0.0.0"
+  "client_addr": "0.0.0.0",
+  "enable_script_checks": false,
+  "disable_remote_exec": true
 }
 ```
 
+~> Since your Consul agent is now available to the network,
+`enable_script_checks` and `remote exec` must be disabled!
```

Comment:
nit: maybe link to the options in docs and be consistent so use `[disable_remote_exec](https://www.consul.io/docs/agent/options.html#disable_remote_exec)` (i,e. the config param rather than the command name? It's a little awkward that one is enable and the other disable but still...


> ~> Since your Consul agent's HTTPS API is available to the network, please make sure [`enable_script_checks`](https://www.consul.io/docs/agent/options.html#_enable_script_checks) is `false` and [`disable_remote_exec`](https://www.consul.io/docs/agent/options.html#disable_remote_exec) is `true`.

Reply:
✔️